runtime/Fortify/fortify.c

```diff
@@ -0,0 +1,88 @@
+//===-- fortify.c ---------------------------------------------------------===//
+//
+//                     The KLEE Symbolic Virtual Machine
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+#include "klee/klee.h"
+
+#include <limits.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <unistd.h>
+
+/* Include fortified versions of the functions defined in the
+   FreeStanding and klee-libc libraries */
+#include "../FreeStanding/fortify.c"
```

Comment:
This should be removed.

Just link those files multiple times.

Reply:
What do you mean?  Add a symbolic link and do `#include "fortify.c"`?  I think it's important not to duplicate the code.